cloudinit/distros/bsd.py

```diff
@@ -51,20 +50,20 @@ def _write_hostname(self, hostname, filename):
         bsd_utils.set_rc_config_value("hostname", hostname, fn="/etc/rc.conf")
 
     def create_group(self, name, members=None):
-        if util.is_group(name):
+        if distros.util.is_group(name):
```

Comment:
We shouldn't be indirectly using modules like this. The `subp` function is defined in `supb.py`, not `distros.py`. I don't think we should apply any of the changes in this file.

What's the reasoning for doing it this way?